src/include/columnar/columnar_tableam.h

```diff
@@ -51,6 +49,9 @@ typedef struct ColumnarScanDescData *ColumnarScanDesc;
 const TableAmRoutine * GetColumnarTableAmRoutine(void);
 extern void columnar_tableam_init(void);
 extern void columnar_tableam_finish(void);
+extern void CreateTruncateTrigger(Oid relationId);
```

Comment:
Looks like this is not referenced or defined in `columnar/`, so I think this line can be removed.